instrumentation/pg/test/opentelemetry/instrumentation/pg/instrumentation_test.rb

```diff
@@ -310,5 +310,25 @@
         _(span.attributes['db.statement']).must_be_nil
       end
     end
+
+    describe '#first_in_list' do
```

Comment:
I see that this is testing a private method. 

Is there some way to reproduce this use case with the `pg` library and check the span attributes instead of invoking the private method directly?

Reply:
Please see my comment on the PR. It's complicated to set up with Docker Compose as it will require an actual connection via a socket to be able to  check the spans.

I gave it a shot for 30 minutes, but ran into different issues with it.